apps/test/unit/weblab/WebLabTest.js

```diff
@@ -12,6 +28,67 @@ describe('WebLab', () => {
     weblab = new WebLab();
   });
 
+  describe('init', () => {
+    let config;
+    beforeEach(() => {
+      stubRedux();
+      stubStudioApp();
+      weblab.studioApp_ = studioApp();
+      registerReducers(commonReducers);
+      registerReducers(reducers);
+      config = {
+        skin: {},
+        level: {}
+      };
+      sinon.stub(ReactDOM, 'render');
+      sinon.stub(getStore(), 'dispatch');
+    });
+
+    afterEach(() => {
+      restoreRedux();
+      restoreStudioApp();
+      ReactDOM.render.restore();
+    });
+
+    it('throws an error if studio app doesnt exist', () => {
+      weblab.studioApp_ = null;
+      expect(weblab.init).to.throw(Error);
+    });
+
+    it('dispatches changeMaxProjectCapacity', () => {
+      weblab.init(config);
+      expect(getStore().dispatch).to.have.been.calledWith(
+        changeMaxProjectCapacity(20971520)
+      );
+      expect(ReactDOM.render).to.have.been.calledOnce;
```

Comment:
i think this was leftover from us pairing 😄 this expectation is unrelated to the test, so we should remove it